ui/app/src/components/ItemStateIcon/ItemStateIcon.tsx

```diff
@@ -152,10 +159,20 @@ export function ItemStateIcon(props: ItemStateIconProps) {
     classes.stateSubmittedToStagingIcon,
     classes.stateSubmittedToLiveIcon,
     classes.stateDisabledIcon,
-    item.stateMap
+    classes.stateStandby,
+    item
   ]);
   return Icon === null ? null : (
-    <Tooltip title={displayTooltip ? getItemStateText(item.stateMap) : ''} open={displayTooltip ? void 0 : false}>
+    <Tooltip
+      title={
+        displayTooltip ? (
+          getItemStateText(item.stateMap)
+        ) : (
+          <FormattedMessage id="words.unknown" defaultMessage="Unknown" />
```

Comment:
if !displayTooltip, then no tooltip should be displayed
```suggestion
          ''
```